entity-management-web/src/integration-test/java/eu/europeana/entitymanagement/web/BaseWebControllerTest.java

```diff
@@ -39,6 +46,16 @@
     @Autowired
     private ObjectMapper objectMapper;
 
+    @Autowired
+    private WebApplicationContext webApplicationContext;
+
+    @BeforeEach
+    protected void setup() throws Exception {
+        this.mockMvc = MockMvcBuilders.webAppContextSetup(this.webApplicationContext).build();
```

Comment:
do we need the server initialization before each test run? This is quite an expensive operation...

Reply:
`MockMvcBuilders.webAppContextSetup()` just creates a new MockMvc instance for each test, which is precisely what we want. It doesn't reload the web application context.